protoc_plugin/CHANGELOG.md

```diff
@@ -1,4 +1,4 @@
-## 22.0.0-wip
```

Comment:
Could you update line 33 in this file with the correct link as well? Not too important. Issue 159 link is broken, we have an unused issue 153 link that should've been link to 159.